app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -888,6 +887,23 @@ impl Node {
                 <+ visualization_visible.not().and(&no_error_set);
 
 
+            // === Bounding Box ===
+
+            let visualization_size             = &model.visualization.frp.size;
+            visualization_enabled_and_visible <- visualization_enabled && visualization_visible;
+            bounding_box_input                <- all4(
+                &new_size,&position,&visualization_enabled_and_visible,visualization_size);
+            out.source.bounding_box <+ bounding_box_input.map(
+                |(node_size,node_position,visualization_enabled_and_visible,visualization_size)| {
+                    Node::node_bounding_box(
+                        *node_size,
+                        *node_position,
+                        *visualization_enabled_and_visible,
+                        *visualization_size,
+                    )
```

Comment:
This is a very special case, because we have all the variable names provided just aline above and everything this does is dereferencing of every variable. In such a case, when all the vars are so clearly visible just a line above, I'd rewrite it to `|(a,b,c,d)| Node::bounding_box(*a,*b,*c,*d)`